src/main/java/com/company/Test.java

```diff
@@ -0,0 +1,14 @@
+package com.company;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class Test {
+
+	public static void main(String[] args) {
+
+		boolean result = StringUtils.isNumeric("Hien");
+		System.out.println(result ? "Day la so.":"Day khong phai la so.");
```

Comment:
Our customer is in the EU. Please never use Vietnamese in the code :)

Reply:
I have adjusted